Exercise_B/index.html

```diff
@@ -13,17 +13,50 @@
       href="https://fonts.googleapis.com/css?family=Lato"
       rel="stylesheet"
     />
-    <!-- Custom Stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
     <link rel="stylesheet" href="./assets/style/stylesheet.css" />
   </head>
 
   <body>
     <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
 
+    <nav class="navbar navbar-light bg-light">
+      <a class="navbar-brand" href="#">
+        <img src='./assets/images/logo.png'  alt="">
+      </a>
+    </nav>
+    <!--Banner Jumbotron of the image-->
+    <div
+  class=" P-10 text-center rounded mb-5 text-white "
+  style="background-image:url(./assets/images/flowershop.jpg); height: 720px;"
+>
+  <h1 class="mb-3 h2">Beautiful Flowers</h1>
+  <button type="button" class="btn btn-success">Learn More</button>
+  
+</div>
     <!--Four containers with img / text-->
     <!-- 4 containers -->

```

Comment:
Do not capitalize HTML elements when we move into React code is will be more clear as to why; For now, know its against conventions to do this.